app/models/git/track.rb

```diff
@@ -69,6 +69,26 @@ def concepts
       config[:concepts].to_a
     end
 
+    memoize
+    def indent_style
+      config[:online_editor][:indent_style].to_sym
+    end
+
+    memoize
+    def indent_size
+      config[:online_editor][:indent_size]
+    end
+
+    memoize
+    def ace_editor_language
+      config[:online_editor][:ace_editor_language]
+    end
+
+    memoize
+    def highlightjs_language
+      config[:online_editor][:highlightjs_language]
+    end
```

Comment:
We should pull out a `online_editor_config` block that the four above reuse, for efficiency?